_episodes/34-code-refactoring.md

```diff
@@ -82,21 +82,25 @@ the tests at all.
 
 > ## Exercise: Write Regression Tests
 > Modify the `analyse_data()` function not to plot a graph and return the data instead.
-> Then, add a new test file called `test_compute_data.py` in the `tests` folder and 
+> Then, adapt the test `test_analyse_data_mock_source.py` in the `tests` folder and 
```

Comment:
From what I'm reading, the file is (still) called `test_compute_data.py`, with `test_analyse_data_mock_source()` being the test function name, see lines 423 and 431 in the previous file.

Reply:
Yes, you are right!